src/main/java/core/basesyntax/model/Draw.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax.model;
+
+public interface Draw {
+    void drawFigure();
```

Comment:
ми можем использовать этот интерфейс не только для фигур
```suggestion
    void draw();
```